bosh-linux-stemcell-builder-master.yml

```diff
@@ -201,7 +204,8 @@ jobs:
           build: versioned-fissile-stemcell-sles
           tag: versioned-fissile-stemcell-sles/VERSION
           build_args:
-            registry: {{docker-internal-registry}}
+            base_image: ((docker-internal-registry))/splatform/os-image-sle12
```

Comment:
Why does this use `((parens))` instead of `{{brackets}}`?

Reply:
My understanding was that `((parens))` is fly-specific, and `{{brackets}}` is more weirdo yaml thing.  Here I used `((parens))` because I am under the impression that it's needed to get string joining semantics (as opposed to whole values).  Is that incorrect?

Reply:
No, `{{brackets}}` is also fly-specific. I've just never seen `((parens))` in a concourse pipeline, so wasn't aware of it at all. Is there any documentation on this?

Having string-joining semantics sounds interesting; I vaguely remember that we wanted this before and created some weird workaround. So I would really like to see/understand how exactly the semantics are defined.

Reply:
It's documented at http://concourse.ci/fly-set-pipeline.html (and in fact `{{brackets}}` isn't).

Reply:
Interesting. I see that `((parens))` are supposed to replace `{{brackets}}` now: https://github.com/concourse/concourse/issues/545

In that case, wouldn't it make sense to change *all* parameter interpolation in the pipeline to use the new format?

Reply:
Yes, but that feels more like a separate PR than this one.